cmd/templ/generatecmd/run/run.go

```diff
@@ -2,39 +2,72 @@ package run
 
 import (
 	"context"
-	"fmt"
 	"os"
 	"os/exec"
+	"runtime"
+	"strconv"
 	"strings"
 	"sync"
+	"syscall"
 )
 
 var m = &sync.Mutex{}
 var running = map[string]*exec.Cmd{}
 
+func KillAll() (err error) {
+	m.Lock()
+	defer m.Unlock()
+	for _, cmd := range running {
+		if runtime.GOOS == "windows" {
+			kill := exec.Command("TASKKILL", "/T", "/F", "/PID", strconv.Itoa(cmd.Process.Pid))
+			kill.Stderr = os.Stderr
+			kill.Stdout = os.Stdout
+			err := kill.Run()
+			if err != nil {
+				return err
+			}
+			continue
+		}
+		err := syscall.Kill(-cmd.Process.Pid, syscall.SIGKILL)
+		if err != nil {
+			return err
+		}
+	}
+	running = map[string]*exec.Cmd{}
```

Comment:
Go 1.21 brings in the new "clear" function, but I want to stick to supporting older versions of Go until they're unsupported by the Go team.

Reply:
So this is good!